helpers/util.sh

```diff
@@ -71,7 +71,7 @@ mkdir -p $LOCAL_REPO
 PLUS_LOCAL_REPO="--plus-repo $LOCAL_REPO"
```

Comment:
Two lines above it sets `LOCAL_REPO=$ANDROID_ROOT/droid-local-repo/$DEVICE` so even when using target for common family of boards, middleware RPMs would still need to somehow be made available for each $DEVICE in its local repo (TMTOWTDI).

Reply:
I changed the local repo to be shared too.